internal/svc/admin_svc.go

```diff
@@ -233,6 +235,35 @@ func (cas *CerbosAdminService) DeleteSchema(ctx context.Context, req *requestv1.
 	return &responsev1.DeleteSchemaResponse{}, nil
 }
 
+func (cas *CerbosAdminService) ReloadStore(ctx context.Context, _ *requestv1.ReloadStoreRequest) (*responsev1.ReloadStoreResponse, error) {
+	log := ctxzap.Extract(ctx)
+	if err := cas.checkCredentials(ctx); err != nil {
+		return nil, err
+	}
+
+	_, err, shared := sfGroup.Do("reload", func() (interface{}, error) {
```

Comment:
Shouldn't this key be different from the key used by the `index.Reload` function? Something like `admin_reload` perhaps?